README.rst

```diff
@@ -26,14 +42,16 @@ normally used in B physics is also provided.
     minuit.migrad()
     unbinned_likelihood.draw(minuit)
 
+Documentation and Tutorial
+--------------------------
 
-* `MIT <http://opensource.org/licenses/MIT>`_ license (open source)
-* `Documentation <http://probfit.readthedocs.org/>`_
+*`Documentation <http://probfit.readthedocs.org/>`_
 * The tutorial is an IPython notebook that you can view online
   `here <http://nbviewer.ipython.org/urls/raw.github.com/scikit-hep/probfit/master/tutorial/tutorial.ipynb>`_.
   To run it locally: `cd tutorial; ipython notebook --pylab=inline tutorial.ipynb`.
-* Dependencies:
-   - `iminuit <http://iminuit.readthedocs.org/>`_
-   - `numpy <http://www.numpy.org/>`_
-   - `matplotlib <http://matplotlib.org/>`_ (optional, for plotting)
```

Comment:
I believe this is currently *not* optional - it really should be made optional, though!

Reply:
Agreed.

Reply:
Am working on that. Will commit soon …

(Aside, @HDembinski: kindly release the tagged 1.1.0 version I prepared yesterday. It addresses wishes by several people ;-). This is now work that is 2nd priority. Thanks a lot.)